### PR TITLE
OSS converted into Open source software

### PR DESCRIPTION
**Pull request by IAMNITESHPANDIT**
some of newbie developer don't understand  meaning of OSS so instead of using OSS just use Open source software it's make your project more familiar with all developers :)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
 			property="og:image"
 			content="https://devprotocol.xyz/assets/img/og.png"
 		/>
-		<meta property="og:description" content="It's Time to Tokenize OSS." />
+		<meta property="og:description" content="It's Time to Tokenize Open source software." />
```
Comment (vinzvinci) [inline public/index.html:55]: Looks good to me, but can you uppercase the s?

Ex.:
- **Open Source Software**

File: public/index.html [hunk 2]
```diff
@@ -313,7 +313,7 @@
 						<div class="col-lg-6">
 							<!-- Info -->
 							<div class="mb-5">
-								<h1 class="text-white mb-5">It's Time to Tokenize OSS</h1>
+								<h1 class="text-white mb-5">It's Time to Tokenize Open source software</h1>
```
Comment (vinzvinci) [inline public/index.html:316]: Same with this, please uppercase the s.

Ex.:
- **Open Source Software**

--- 
Thank you for contributing <3

File: public/index.html (hunk 1 above)
Comment (IAMNITESHPANDIT) [reply public/index.html:55]: Yes :)

**Comment by vinzvinci** [comment]
Any update?

File: public/index.html (hunk 1 above)
Comment (vinzvinci) [reply public/index.html:55]: How about this line

Comment (IAMNITESHPANDIT) [reply public/index.html:55]: It would be enhance organic traffic  on your website with the help SEO .

**Comment by vinzvinci** [comment]
Can you also check this @kazu80?

**Review by kazu80 — APPROVED** [review]

**Comment by kazu80** [comment]
Thanks !!!  I merged it

**Pull request merged**